src/schema.sql

```diff
@@ -69,28 +83,27 @@ CREATE TABLE geo (
 CREATE INDEX geo_gist ON geo USING GIST(geom);
 CREATE INDEX geo_idx ON geo(id);
 
-DROP TABLE IF EXISTS styles;
+CREATE TABLE geo_history (
+    id          BIGINT NOT NULL,
+    delta       BIGINT NOT NULL REFERENCES deltas(id),
```

Comment:
I understand the application of the reference here, but I'd like to know more about what we're trying to protect against by adding it. The constraint will add time to inserts, etc...

Reply:
Removed in https://github.com/mapbox/Hecate/pull/206/commits/7493e0efafaeec0af7b70cbab56df3f1b16525d5 -- while these are changes we should make, i'm removing them from the current PR to reduce scope for what will already be a large change.